roles/openshift_logging_eventrouter/templates/eventrouter-dc.j2

```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+kind: DeploymentConfig
+metadata:
+  name: logging-eventrouter
+  labels:
+    component: eventrouter
+    logging-infra: eventrouter
+    provider: openshift
+spec:
+  selector:
+    component: eventrouter
+    logging-infra: eventrouter
+    provider: openshift
+  replicas: "{{ replicas }}"
+  template:
+    metadata:
+      labels:
+        component: eventrouter
+        logging-infra: eventrouter
+        provider: openshift
+      name: logging-eventrouter
+    spec:
+      serviceAccount: aggregated-logging-eventrouter
+      serviceAccountName: aggregated-logging-eventrouter
+{% if node_selector is iterable and node_selector | length > 0 %}
+      nodeSelector:
+{% for key, value in node_selector.iteritems() %}
+        {{ key }}: "{{ value }}"
+{% endfor %}
+{% endif %}
+      containers:
+        - name: kube-eventrouter
+          # TODO: reference rh registry once it gets there
+          image: gcr.io/heptio-images/eventrouter:v0.1
```

Comment:
This should become variables of image and version so can change them accordingly for upstream and downstream, etc